packages/graph/hash_graph/lib/interval-ops/src/interval.rs

```diff
@@ -183,7 +183,7 @@ pub trait Interval<T>: Sized {
         // Range A:   [-----] | [-----]   | [-----]         |         [-----] | [---------]
         // Range B: [-----]   |   [-----] |         [-----] | [-----]         |   [-----]
         // Result:  [-------] | [-------] | [-------------] | [-------------] | [---------]
```

Comment:
This confused me for quite a while, can you add a heading above each column to explain they're examples or something (e.g. `Example 1 | Example 2`, or maybe just a top level heading saying "Each column constitutes an example"

Reply:
Thanks, I'll respond there